photon-ml/src/main/scala/com/linkedin/photon/ml/evaluation/RMSEEvaluator.scala

```diff
@@ -26,7 +26,7 @@ import org.apache.spark.rdd.RDD
   */
 protected[ml] class RMSEEvaluator(
     labelAndOffsetAndWeights: RDD[(Long, (Double, Double, Double))],
-    defaultScore: Double = 0.0) extends Evaluator {
+    defaultScore: Double = 0.0) extends Evaluator(RMSE) {
```

Comment:
Isn't this the ```Evaluator``` trait? How are we passing constructor arguments here?

I think it should be like the others:

```scala
    defaultScore: Double = 0.0) extends Evaluator {

    protected val evaluatorType = RMSE
```